CHANGELOG.md

```diff
@@ -1,4 +1,34 @@
-# v2.4.1 (unreleased)
+# v2.5.0 (unreleased)
+### Notes
+- This release supports OneView Rest API versions 800/1000/1200 for Hypervisor Resources.
+
+### Oneview Features supported
```

Comment:
Please add only the resource which is new supported for this release. ie. Hypervisor Cluster Profile in this case

Reply:
done